ground/gcs/gcs.pri

```diff
@@ -67,6 +67,7 @@ macx {
     GCS_DATA_BASENAME = Resources
     GCS_DOC_PATH     = $$GCS_DATA_PATH/doc
     GCS_BIN_PATH     = $$GCS_APP_PATH/$${GCS_APP_TARGET}.app/Contents/MacOS
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9

```

Comment:
It lets us use modern language features.  C++11 is a pretty big win.  And the proportion of users on older releases continues to fall.  Note that the QT guys have already dropped official support for 10.7 in 5.5 and have indicated 10.8 will probably go soon.

constexpr and initializer lists let us do a lot more work at compile time instead of startup and runtime.  It also makes a lot of code more compact (removing millions of .append(constant) lines, for instance).  It improves compatibility with C code.  There's an auto keyword that can infer return types so that you can handle several kinds of lists with the same code.  And it catches more bugs at compile time.

I'm working pretty deeply on the uavobjectgenerator,